webpack.common.config.ts

```diff
@@ -48,7 +48,11 @@ module.exports = {
     extensions: ['.ts', '.js', '.json', '.html', '.svg', '.png'],
     alias: {
       '@angular/upgrade/static': (
-        '@angular/upgrade/bundles/upgrade-static.umd.js')
+        '@angular/upgrade/bundles/upgrade-static.umd.js'),
+      // This is needed because in app.constans.ts we need to import
```

Comment:
constants is misspelled here and elsewhere

Reply:
done